### PR TITLE
Add spirit emblems.

### PR DESCRIPTION
**Pull request by tomprince**

### CODE REVIEW

**Comment by iakona** [comment]
none of those objects should have grid or snap turned on

File: objects/spiritEmblems/object.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  "Description": "",
+  "GMNotes": "",
+  "ColorDiffuse": {
+    "r": 0.567665,
+    "g": 0.560111,
+    "b": 0.5600328
+  },
+  "LayoutGroupSortIndex": 0,
+  "Locked": true,
+  "Grid": true,
```
[the remark this answers is not in the document]
Comment (tomprince) [reply objects/spiritEmblems/object.json:24]: Hmm. At least some of the other locked bags there have both of those set to true (I copied this definition from one of them).

[the remark this answers is not in the document]
Comment (iakona) [reply objects/spiritEmblems/object.json]: sure things already in the mod are not tagged appropriately, and it's not really worth the effort to go back and fix all of them, but anything new getting added into the mod should be correct

[the remark this answers is not in the document]
Comment (tomprince) [reply objects/spiritEmblems/object.json]: (Note that I've update the bag already)

**Comment by iakona** [comment]
these were eventually added in

**Pull request closed without merging**